src/provisioner/const.py

```diff
@@ -25,3 +25,5 @@
 # 'UPGRADE_MODE_VAL' constant for default upgrade mode.
 UPGRADE_MODE_VAL = ""
 REQUIRED_EXTERNAL_SW = ['kafka', 'consul']
+CORTX_DELTA_URL = "yaml:///etc/cortx/delta.conf"
+TMP_CORTX_CONF_URL = "yaml:///tmp/tmp.conf"
```

Comment:
TMP need not be declared here unless two distinct module wants to share this

Reply:
removed TMP_URL from const.py and added in provisioner.py